applications/endoscopy_tool_tracking/python/endoscopy_tool_tracking.py

```diff
@@ -23,27 +25,41 @@
     VideoStreamRecorderOp,
     VideoStreamReplayerOp,
 )
-
-# Uncomment UnboundedAllocator to enable DELTACAST capture card (linter issue)
-from holoscan.resources import (  # UnboundedAllocator,
+from holoscan.resources import (
     BlockMemoryPool,
     CudaStreamPool,
     MemoryStorageType,
+    UnboundedAllocator,
 )
 
-from holohub.aja_source import AJASourceOp
 from holohub.lstm_tensor_rt_inference import LSTMTensorRTInferenceOp
-
-# Enable this line for Yuam capture card
-# from holohub.qcap_source import QCAPSourceOp
 from holohub.tool_tracking_postprocessor import ToolTrackingPostprocessorOp
 
-# Uncomment to enable DELTACAST capture card
-# from holohub.videomaster import VideoMasterSourceOp, VideoMasterTransmitterOp
 
+def lazy_import(module_name):
+    """Lazily import a module by name.
+
+    This function provides a way to import modules only when they are first accessed,
+    which can help reduce startup time and memory usage. It uses Python's importlib
+    to handle the lazy loading.
+
+    Args:
+        module_name (str): The name of the module to import
 
-# Enable this line for vtk rendering
-# from holohub.vtk_renderer import VtkRendererOp
+    Returns:
+        module: The imported module object
+
+    Raises:
+        ImportError: If the specified module cannot be found
+    """
+    spec = importlib.util.find_spec(module_name)
+    if spec is None:
+        raise ImportError(f"Module {module_name} not found")
+    module = importlib.util.module_from_spec(spec)
+    loader = importlib.util.LazyLoader(spec.loader)
+    loader.exec_module(module)
+    sys.modules[module_name] = module
+    return module
```

Comment:
Oh that's cool, I didn't know about `LazyLoader`. This is what I did back then for the SDK, which works fine but using LazyLoader could be helpful too: https://github.com/nvidia-holoscan/holoscan-sdk/blob/main/python/holoscan/__init__.py.in#L103-L133